src/test/java/plugins/LdapPluginTest.java

```diff
@@ -73,6 +90,24 @@ public void login_ok() {
     }
 
     /**
+     * Scenario: Login with ldap uid and password
+     * Given I have a docker fixture "ldap" that allows anonymous binding
+     * And Jenkins is using ldap as security realm
+     * When I login with user "jenkins" and password "root" without manager credentials
+     * Then I will be successfully logged on as user "jenkins"
```

Comment:
Thanks for the info Oliver. Nothing is mentioned in the Wiki:

> If your LDAP server doesn't support anonymous binding (IOW, if your LDAP server doesn't even allow a query without authentication), then Jenkins would have to first authenticate itself against the LDAP server, and Jenkins does that by sending "manager" DN and password.

And I obviously wasn't aware of that. Test case removed. Again, thanks for the heads up.